cloudify_handler/cloudify.py

```diff
@@ -80,8 +82,16 @@ def process(self, metric):
                 routing_key=metric.getPathPrefix(),
                 body=jsonify(metric))
 
+            consec_err_count = 0
         except Exception:  # Rough connection re-try logic.
-            self.log.info(
-                "Failed publishing to rabbitMQ. Attempting reconnect")
+            if consec_err_count == 0:
+                self.log.info(
+                    "Failed publishing to rabbitMQ. Attempting reconnect")
+            # Reducing 90% of errors in logs when
+            # there is a lasting connection problem
+            elif consec_err_count == 9:
+                consec_err_count = 0
+
+            consec_err_count = consec_err_count + 1
             self._bind()
             sleep(3)
```

Comment:
btw maybe instead 3 we can make it longer - using exponential backoff, so it's first sleep(1), then 2, then 4, 8.. up to 30
like here https://github.com/cloudify-cosmo/cloudify-common/blob/master/cloudify/amqp_client.py#L107-L113

Reply:
Agreed- also, that might mean that the complexity of the consec_err_count logic isn't needed?